vtm.js

```diff
@@ -326,4 +476,122 @@ app.component('stat', {
+        />
+      <b>Age:</b>
+      <input
+        :style= "{width: 7 + 'ch'}"
+        v-model="bio.age"
+        type="number"
+        :placeholder = '123'/>
+      <b>Generation:</b>
+      <select 
+        v-model="bio.generation">
+        <option disabled value=null>-none-</option>
+        <option>15th</option>
+        <option>14th</option>
+        <option>13th</option>
+      </select>
+      <b>Clan:</b>
+      <select v-model="bio.clan">
+        <option disabled value="">Pick a Clan</option>
+        <option 
+          v-for="clan in clans"
+          :key = "clan.id"
+          :value="clan">
+          {{clan.id}}
+        </option>
+      </select>
+      <b>Faction:</b>
+      <select v-model="bio.faction">
+        <option disabled value="">Choose Your Coterie's Faction</option>
+        <option>Cammarilla</option>
+        <option>etc</option>
+        <option>C</option>
+      </select>
+      </div>`
+});
+
+app.component('discipline-section', {
+  mixins:[statSectionMixin],
+  props:['clan'],
+  computed: {
+    /**
+     * @returns {Array} of numbers describing how many points are currently assigned
+     * for example [0,9,0,0,0,0] means there are 9 attributes with value 1
+     * while [0,1,4,3,1,0] means 1 attribute with value 1, 4 attributes with value 2,
+     * 3 attributes with value 3, 1 attribute with value 4 and 0 attributes with value 5
+     */
+    allocatedResources() {
+      let tmp = Array(this.stats.resource.length).fill(0);
+      for (var i = 0; i < this.stats.data.length; i++) {
+        tmp[this.stats.data[i].value]++;
+      }
+      return tmp;
+    },
+  },
+  methods:{
+    /**
+     * @returns if discipline is primary or not
+     */
+    isPrimary(discipline){
+      if (!this.clan.abilities) return false;
+      for (var i = 0; i< this.clan.abilities.length; i++){
+        if (discipline.id == this.clan.abilities[i]){
+          return true;
+        }
+      }
+      return false;
+    },
+  },
+  template: `
+    <div class="statSection">
+      <h2>{{stats.id}}</h2>
+      <div class="resourceCount">{{allocatedResources}} | {{stats.resource}}</div>
+      <div class="statList">
+        <h2>Primary</h2>
+        <ul class="ulStats">
+          <li 
+            v-for="item in stats.data"
+            :key="item.id">
+            <stat 
+              v-show = "isPrimary(item)"
+              :stat="item"
+              :scale="stats.resource.length - 1"
+              @stat-change="emitAllowedChange($event)">
+            </stat>
+          </li>
+        </ul>
+      </div>
+      <div class="statList">
+        <h2>Secondary</h2>
+        <ul class="ulStats">
+          <li 
+            v-for="item in stats.data"
+            :key="item.id">
+            <stat 
+              v-show = "!isPrimary(item)"
+              :stat="item"
+              :scale="stats.resource.length - 1"
+              @stat-change="(!clan.abilities || $event[2]) ? emitAllowedChange($event) : ''">
```

Comment:
Secondary by sa nemali menit? Alebo mali by sa menit len v pripade, ze je to Caitiff?

Reply:
secondary sa nemenia pokial, je to add tnz `$event[2]==true` ale v pripade ze nema clan.abilities tzn Caitiff, Thin Blood

Reply:
premyslam, ci by tam niekde nesiel dat maly komentar, aby to bolo citelnejsie, ale neviem kam. Tak to asi nechame tak